pkg/events/events.go

```diff
@@ -74,10 +76,46 @@ type Event struct {
 	Params       []trace.ArgMeta
 }
 
+// NewEventDefinition creates a new event definition
+func NewEventDefinition(name string, sets []string, depsID []ID) Event {
+	evt := Event{
+		ID32Bit: sys32undefined,
+		Name:    name,
+		Sets:    sets,
+	}
+
+	d := dependencies{
+		Events: make([]eventDependency, 0, len(depsID)),
+	}
+
+	for _, id := range depsID {
+		d.Events = append(d.Events, eventDependency{EventID: id})
+	}
+
+	evt.Dependencies = d
+
+	return evt
+}
+
 type eventDefinitions struct {
 	events map[ID]Event
 }
 
+// Add add event to defitions
```

Comment:
Perhaps a comment on `var Definitions = eventDefinitions{` to make it clear that this is map that changes at runtime?